README.md

```diff
@@ -25,6 +25,11 @@ Currently supported distros for ansible jump host are:
 
 * > RHEL 7.7
 
+# Unsupported machine types
```

Comment:
if the user adds the interface name in the all.yaml, it will work, right? if so you can mention that instead of saying just unsupported. so it will be helpful.

Reply:
done